chpl/chpl-service/src/main/java/gov/healthit/chpl/CHPLServiceConfig.java

```diff
@@ -169,6 +182,15 @@ public LocaleChangeInterceptor localeInterceptor() {
         return interceptor;
     }
 
+    @Bean
+    public InternalResourceViewResolver viewResolver() {
```

Comment:
Do we need this?

Reply:
I honestly do not know...  This was pulled over from the AppConfig.java which was deleted.  This was the only bean in that config file that was not in CHPLServiceConfig.java., so I thought I should bring it over just in case.  I think we need to look at our Spring config files - there are 4 or so different files and I think we can combine them.